### DISCUSSION TITLE
Can I use existing API to add *real* footnotes?

### DISCUSSION THREAD

**Discussion by ciacob** [dolanmiu/docx#1154] [General]
Can I use existing API to add *real* footnotes? Where, by *real* footnotes I refer to auto-numbered text that lives in a reserved, special section of each page's footer and consumes a portion of the rest of the text on the page. Most importantly, this text reflows as needed (moves from page to page), as the document changes and grows.

**Comment by ciacob** [comment]
Nevermind that, I actually just found a "Footnotes" example. But I don't recall to have spotted any reference to the "footnotes" property in the documentation. I will look again.

**Reply by dolanmiu** [reply]
Documentation is not too complete. feel free to create some documentation if you'd like!
